modules/ROOT/pages/dynamic-evaluate-component-reference.adoc

```diff
@@ -21,7 +21,158 @@ The script can use any of the usual xref:dataweave-variables-context.adoc[contex
 
 |===
 
-== Example XML Configuration
+== Evaluate DataWeave Script from a File System Example
+
+The following example shows that based on a POST request made to the Mule application, the Dynamic Evaluate Component selects and executes a DataWeave script stored locally in the file system. The application's flow behavior is as follows:
+
+* An HTTP Listener source listens to a POST URL request that contains a locale query parameter and a payload.
```

Comment:
locale > `locale` or _locale_ to clarify that this is the parameter's name. Apply this change in all places where "locale" appears.